roles/vector/molecule/debian12/molecule.yml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
why are the Molecule tests inside the role?
I usually place then in the collection root and outside any implementation. Is there s specific reason for doing it this way?

Reply:
Referring to the site https://www.ansible.com/blog/developing-and-testing-ansible-roles-with-molecule-and-podman-part-1/ it seems that the correct place for molecule directory is inside the role directory. On this site https://ansible.readthedocs.io/projects/molecule/getting-started/ the molecule folder is outsite the roles path. I think there is no best practise.

Reply:
hmm, I'd really like to move the molecule files outside the role. It seems way cleaner to me.

Reply:
Done with Commit https://github.com/telekom-mms/ansible-vector/pull/2/commits/2f51d0841ea6563c72c7b29eec8c456adfbdf982.

Reply:
changed, need to fix rockylinux8 test